src/serializers/index.ts

```diff
@@ -77,14 +80,26 @@ const res = (response: Response) =>
       }
     : response;
 
-export const createSerializers = (opts: SerializerOptions) => {
+const createErrSerializer =
+  (
+    serializer: (error: Error) => SerializedError,
+    opts: FormatterOptions,
+  ): SerializerFn =>
+  (error: unknown): unknown =>
+    trimmer({
+      depth: opts.maxObjectDepth ?? 4,
```

Comment:
1. Nitpick: can we set the default (4) in one spot?
2. Could some engineers be relying on the fact that they can read deeply nested error information, e.g. to review a HTTP request and response body and figure out what caused a HTTP 400? Not to say we should encourage this practice, but it may be an easy out that is available today, and I wonder if `4` is still a reasonable global default in this light.

   On the topic of release, I'd almost err toward a major version here.